Charts/Classes/Renderers/CandleStickChartRenderer.swift

```diff
@@ -43,7 +43,7 @@ public class CandleStickChartRenderer: LineScatterCandleRadarChartRenderer
         }
     }
     
-    private var _shadowPoints = [CGPoint](repeating: CGPoint(), count: 4)
+    private var _shadowPoints = [CGPoint](repeating: CGPoint(), count: 2)
```

Comment:
such changes should be included I guess, if not converted by Xcode 8. I would only use the conversion tool to do the job, since it's all about swift 3.0, not about bugs or issues. If you manually changed something, with so many diff in one PR, it's hard to find out.

Anyway, swift 3.0 branch is just for testing